packages/pnpm/spnpm.js

```diff
@@ -1,3 +1,60 @@
-require('@pnpm/ts-execution-runtime')
```

Comment:
I guess this project should be removed then.

Reply:
Sounds good. I can do that. I wasn't sure if it was used elsewhere.